### ISSUE TITLE
.rubocop.yml exclude doesn't work.

### ISSUE THREAD

**Issue by ghost** [rubyide/vscode-ruby#359]
### Your environment

- `vscode-ruby` version: 0.18.0
- Ruby version: ruby 2.4.4p296 (2018-03-28 revision 63013) [x64-mingw32]
- VS Code version: 1.24.1 and 1.25.0-insider 
- Operating System: Windows 10 latest

### Expected behavior

`.rubocop.yml` like this should work:

```
AllCops:
  TargetRubyVersion: 2.5
  Exclude:
    - 'db/**'
    - '**/Gemfile'
    - 'Gemfile.lock'
```

### Actual behavior

`db/schema.rb`, `Gemfile` and `Gemfile.lock` being rubocopped.

Via `rubocop db` it works. So, **it's a bug**?

**Comment by bowmanmike** [comment]
Any update here? Would love to get this issue resolved.

I have `db/schema.rb` excluded in my `.rubocop.yml` but VSCode still tries to lint it. My schema file is huge (10k lines), and doesn't conform to our style guide, so there's tons of errors. Just opening the file cranks my CPU usage up to 100% for that process, and I have to kill it manually.

**Comment by MiralDesai** [comment]
I'm just moving over from Atom and noticed this. The readme says the includes and excludes will not work in any of the linters which doesn't sound good. Same thing is happening with Reek where I've disabled it for a folder and it still runs.

**Comment by wingrunr21** [comment]
This will be fixed via #405

**Comment by wingrunr21** [comment]
#405 has been merged
